regulations/static/regulations/css/scss/comment/_comment.scss

```diff
@@ -44,40 +44,40 @@ N&C Header - including Read/Write tab
     width: 245px;
 
     a {
-      color: $gray_darker;
+      color: $panel_text_color;
       padding-left: 5px;
     }
 
     .read-icon,
     .write-icon {
       padding: 0;
       font-size: 14px;
-      fill: $gray_darker;
+      fill: $panel_text_color;
     }
 
     &.active-mode a {
-      color: $blue;
+      color: $primary_color;
     }
 
     &:hover span,
     &:hover a {
-      color: $blue;
+      color: $primary_color;
     }
 
     &:hover:not(.active-mode) {
-      background: lighten($gray, 10%);
+      background: $white;
```

Comment:
as far as color translations go I think the color computed by the fuction used to be somewhere between the lightest gray we used and white which works pretty well with the blues we are using but potentially wouldn't work as well if someone were to override `$action_color` with something wildly different. I decided to just revert to white to retain consistency with the rest of the app where `$action_color` always sits on a white background, ensuring that the hover state for this one tab wouldn't create a contrast issue unique to this one element. If that makes sense.